src/ORM/FieldType/DBObjectType.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace DNADesign\Elemental\ORM\FieldType;
+
+use SilverStripe\GraphQL\Manager;
+use SilverStripe\ORM\FieldType\DBField;
+
+class DBObjectType extends DBField
+{
+    /**
+     * Add the field to the underlying database.
+     */
+    public function requireField()
+    {
+        // I am mad that I have to add this.
```

Comment:
???

Reply:
Yes, it was experimental when I was writing this :joy:. Should be replaced with something like `// noop - This class exists only to bind to a custom GraphQL type`.